### PR TITLE
SONAR-7175 DbCleaner keeps the oldest analysis instead of the newest one

### PR DESCRIPTION
**Pull request by julienlancelot**

### CODE REVIEW

File: travis.sh [hunk 1]
```diff
@@ -28,7 +28,7 @@ CI)
         -Pdeploy-sonarsource \
         -B -e -V
 
-  elif [[ "${TRAVIS_BRANCH}" == "branch-"* ]] && [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
```
Comment (teryk) [inline travis.sh]: do not push in master : )

Comment (julienlancelot) [inline travis.sh]: Indeed, that's why it's in the name of the commit !

**Comment by teryk** [comment]
LGTM

**Comment by teryk** [comment]
thank you for the scenario to reproduce the bug

**Comment by julienlancelot** [comment]
Thanks ! And for the scenario to reproduce, it was hard to find, but I'm proud to find it !

**Pull request merged**